src/CP/valueselection/learning/learnedheuristic.jl

```diff
@@ -33,8 +33,8 @@ mutable struct LearnedHeuristic{SR<:AbstractStateRepresentation, R<:AbstractRewa
     current_state::Union{Nothing, SR}
     reward::Union{Nothing, R}
     search_metrics::Union{Nothing, SearchMetrics}
-
-    LearnedHeuristic{SR, R, A}(agent::RL.Agent) where {SR, R, A}= new{SR, R, A}(agent, nothing, nothing, nothing, nothing, nothing, nothing)
+    firstActionTaken::Bool
```

Comment:
new attribute to avoid using the non RL-related statistic : `model.statistics.numberOfNodes` to check if one action was already made by the agent.